vale/vocab.txt

```diff
@@ -0,0 +1,137 @@
+addr
+api
+apis
+architecting
+artifactory
+Artifactory
+Atlassian
+autogenerated
+autoscaling
+Autoscaling
+Basecamp
+Basscss
+Bazel
+Bitium
+boolean
+boomper
+Boomper
+Buildbox
+buildkite
+Buildkite
+buildscript
+chatbot
+chroot
+chroots
+cli
+cloudformation
+Cloudwatch
```

Comment:
These should probably be CloudWatch and CloudFormation (though the all-lower variant might be what we want somewhere?)